docs/howto/queryable-encryption.rst

```diff
@@ -0,0 +1,331 @@
+================================
+Configuring Queryable Encryption
+================================
+
+.. versionadded:: 5.2.3
+
+:doc:`manual:core/queryable-encryption` is a powerful MongoDB feature that
+allows you to encrypt sensitive fields in your database while still supporting
+queries on that encrypted data.
+
+This section will guide you through the process of configuring Queryable
+Encryption in your Django project.
+
+.. admonition:: MongoDB requirements
+
+    Queryable Encryption can be used with MongoDB replica sets or sharded
+    clusters running version 8.0 or later. Standalone instances are not
```

Comment:
Users may wonder why it says 8.0 when the MongoDB documentation says 7.0. Even if we only test with 8 because of the range/rangePreview issue, I think our implementation should work on 7.